### PR TITLE
Add new integration nexia

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add new integration nexia

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/32826
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/nexia.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Air Cleaner Mode
+- Current Compressor Speed
+- Requested Compressor Speed
+- Outdoor Temperature
+- Relative Humidity
+- System Status
+
+The following binary sensors are added for each thermostat zone:
+
+- Zone Tempature
```
Comment (frenck) [inline source/_integrations/nexia.markdown]:
```suggestion
- Zone Temperature
```

File: source/_integrations/nexia.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- Zone Setpoint Status
+- Zone Status
+
+### Climate
+
+The `nexia` climate platform lets you control a thermostat
+
+
```
Comment (frenck) [inline source/_integrations/nexia.markdown]:
```suggestion

```

File: source/_integrations/nexia.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+The following binary sensors are added for each thermostat zone:
+
+- Zone Tempature
+- Zone Setpoint Status
+- Zone Status
+
+### Climate
+
+The `nexia` climate platform lets you control a thermostat
```
Comment (frenck) [inline source/_integrations/nexia.markdown]:
```suggestion
The `nexia` climate platform lets you control a thermostat.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small things... Other than that: 👍

**Comment by bdraco** [comment]
Thanks @frenck

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request merged**